pkg/commands/workload.go

```diff
@@ -517,7 +518,7 @@ func (opts *WorkloadOptions) Update(ctx context.Context, c *cli.Config, currentW
 
 	if msgs := workload.DeprecationWarnings(); len(msgs) != 0 {
 		for _, msg := range msgs {
-			c.Infof("WARNING: %s\n", msg)
+			c.Emoji(cli.Exclamation, cliprinter.Sinfof("WARNING %s\n", msg))
```

Comment:
Is this a design decision to remove the `:` after `WARNING` ? update command deprecation message is different from this. Inconsistent output